app/model/Forms.scala

```diff
@@ -307,5 +309,14 @@ object SendEmailPrefencesChoiceForm {
 
     override def unbind(key: String, value: EmailPreferencesChoice) = Map(key -> value.toString)
   }
-}
 
+  final case class UpdateGrantLengthForm(grantLength: Option[Int])
+
+  object UpdateGrantLengthForm {
+    val form: Form[UpdateGrantLengthForm] = Form(
+      mapping(
+        "grantLength" -> optional(number).verifying("grantLength.required", _.isDefined)
```

Comment:
Looks odd to be using optional and then verifying it's defined.  Might be wrong but it looks odd.

Reply:
Its a bit strange to me too that grantLength field isn't mapping to a non-optional number at runtime. Later I noticed that wherever radio multiselect option field is used, the form validation takes optional value in its mapping.

Reply:
Ah.  I don't remember that at all from writing forms.  I now want to play around with this.  Seems really inconsistent.